### PR TITLE
Mock endpoints

### LINKED ISSUE #2041: Add "mock" version endpoints to V2 augur-node
1 . Account Summary (src/server/getters/get-account-time-ranged-stats.ts)
2. System Status getters (src/server/getters/get-platform-activity-stats.ts)
3. Transaction history (src/server/getters/get-account-transaction-history.ts)

### PR DESCRIPTION
**Pull request by adrake33**
Fixes https://github.com/AugurProject/augur/issues/2041

Add mock endpoints for `getFeeWindow`, `getFeeWindowCurrent`, `get-account-time-ranged-stats`, `get-platform-activity-stats`, & `get-account-transaction-history`.

### CODE REVIEW

File: packages/augur-artifacts/src/addresses.json [hunk 1]
```diff
@@ -175,19 +175,19 @@
   "TradingEscapeHatch": "0xdfef677bf5f66f3eeff481a587c04cb58e95b92a"
  },
  "12346": {
-  "Augur": "0x3730Ea866Aa1709eD4097Cb1a92F817AD2DC7c56",
-  "Universe": "0x09369758c16F85D7d91bE219Fe40048dddf0EAeC",
```
Comment (justinbarry) [inline packages/augur-artifacts/src/addresses.json]: Why did these change?

**Review by justinbarry — CHANGES_REQUESTED** [review]

File: packages/augur-artifacts/src/addresses.json (hunk 1 above)
Comment (adrake33) [reply packages/augur-artifacts/src/addresses.json]: That happened when I rebuilt for some reason. I'll revert these.

**Comment by bthaile** [comment]
looks like getMarkets doesn't like CROWDSOURCING_DISPUTE or AWAITING_FORK_MIGRATION

image

**Comment by bthaile** [comment]
column doesn't exist error. `markets.marketCreatorFeeRate`

image

**Comment by bthaile** [comment]
addresses.json change, need to totally revert changes, or just delete all from 12346 networkId all all those timestamp networks same for upload-block-numbers.json

**Review by justinbarry — APPROVED** [review]

**Pull request merged**